index.js

```diff
@@ -68,7 +73,36 @@ function toggleColor(element) {
   }
 }
 
+element.addEventListener("click", function() {
+  // console.log("hi")
+  // console.log(toggleColor)
+  toggleColor(element)
+})
+
+
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector(".like-button")
 
+likeButton.addEventListener("click", function() {
+  // console.log("liked")
+  const likes = document.querySelector('.likes')
+  const likesNumber = parseInt(likes.textContent.split(' ')[0]) + 1
+  // debugger
+  likes.textContent = `${likesNumber} ${likes.textContent.split(' ')[1]}`
```

Comment:
Looks good! The key here is that you have to _read_ some information from the DOM to get the current number of likes, increment it, and then _write_ it back to the DOM. One tip - `parseInt` will strip any non-number characters from the _end_ of a string, so you could just do:

```js
  const likes = document.querySelector('.likes')
  const likesNumber = parseInt(likes.textContent) + 1
  likes.textContent = `${likesNumber} Likes`
```